_includes/codepens/inline/index.html

```diff
@@ -1,62 +1,25 @@
-      <td class="demo-inline-content" style="padding-bottom: 0">
-        <img src='https://www.tiny.cloud/images/email-banner-2.png' style="width:650px" />
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <table>
-          <tbody>
-            <tr>
-              <td class="demo-inline-content" style="padding-bottom: 10px">
-                <h2 class="tinymce-heading">The world’s first rich text editor in the cloud</h2>
-                <div class="tinymce-body">
-                  <p>
-                    Have you heard about Tiny Cloud? 
-                    It’s the first step in our journey to help you deliver great content creation experiences, no matter your level of expertise. 
-                    50,000 developers already agree.
-                    They get free access to our global CDN, image proxy services and auto updates to the TinyMCE editor. 
-                    They’re also ready for some exciting updates coming soon.
-                  </p>
-                  <p>
-                    One of these enhancements is <strong>Tiny Drive</strong>: imagine file management for TinyMCE, in the cloud, made super easy. 
-                    Learn more at <a href='https://www.tiny.cloud/tinydrive/'>tiny.cloud/tinydrive</a>, where you’ll find a working demo and an opportunity to provide feedback to the product team.
-                  </p>
-                </div>
-              </td>
-              <td class="demo-inline-content">
-                <img style="width:300px;max-width:300px;" src='https://www.tiny.cloud/images/image-two.jpg'/>
-              </td>
-            </tr>
-          </tbody>
-        </table>
-      </td>
-    </tr>
-    <tr>
-      <td class="demo-inline-content" style="padding-top: 0">
-        <h3 class="tinymce-heading">An editor for every project</h3>
-        <div class="tinymce-body">
-          <p>
-            Here are some of our customer’s most common use cases for TinyMCE:
-          </p>
-          <ul>
-            <li>Content Management Systems (<em>e.g. WordPress, Umbraco</em>)</li>
-            <li>Learning Management Systems (<em>e.g. Blackboard</em>)</li>
-            <li>Customer Relationship Management and marketing automation (<em>e.g. Marketo</em>)</li>
-            <li>Email marketing (<em>e.g. Constant Contact</em>)</li>
-            <li>Content creation in SaaS systems (<em>e.g. Eventbrite, Evernote, GoFundMe, Zendesk</em>)</li>
-          </ul>
-        
-          <p>
-            And those use cases are just the start. 
-            TinyMCE is incredibly flexible, and with hundreds of APIs there’s likely a solution for your editor project. 
-            If you haven’t experienced Tiny Cloud, get started today. 
-            You’ll even get a free trial of our premium plugins – no credit card required!
-          </p>
-        </div>
-      </td>
-    </tr>
-  </tbody>
-</table>
+<div class="demo-inline">
+  <div style="background-color: #fafafa; padding: 20px 20px 10px 20px;">
```

Comment:
Just a thought, but maybe instead of using inline styles here, use a class and then add these styles to the codepen css page, to show how it uses the site css for inline mode?

Reply:
It looks as though we are actually doing that for a few things, so I think this style at least should be moved over there.

Reply:
moved